tests/src/test/java/org/neo4j/graphalgo/algo/EigenvectorCentralityProcIntegrationTest.java

```diff
@@ -0,0 +1,299 @@
+            db.execute("LOAD CSV WITH HEADERS FROM 'file:///got-s1-nodes.csv' AS row\n" +
+                    "MERGE (c:Character {id: row.Id})\n" +
+                    "SET c.name = row.Label;").close();
+
+            db.execute("LOAD CSV WITH HEADERS FROM 'file:///got-s1-edges.csv' AS row\n" +
+                    "MATCH (source:Character {id: row.Source})\n" +
+                    "MATCH (target:Character {id: row.Target})\n" +
+                    "MERGE (source)-[rel:INTERACTS_SEASON1]->(target)\n" +
+                    "SET rel.weight = toInteger(row.Weight);").close();
+
+            tx.success();
+        }
+
+        Procedures procedures = db.getDependencyResolver().resolveDependency(Procedures.class);
+        procedures.registerProcedure(EigenvectorCentralityProc.class);
+        procedures.registerProcedure(PageRankProc.class);
+
+
+        try (Transaction tx = db.beginTx()) {
+            final Label label = Label.label("Character");
+            expected.put(db.findNode(label, "name", "Ned").getId(), 111.68570401574802);
+            expected.put(db.findNode(label, "name", "Robert").getId(), 88.09448401574804);
+            expected.put(db.findNode(label, "name", "Cersei").getId(), 		84.59226401574804);
+            expected.put(db.findNode(label, "name", "Catelyn").getId(), 	84.51566401574803);
+            expected.put(db.findNode(label, "name", "Tyrion").getId(), 82.00291401574802);
+            expected.put(db.findNode(label, "name", "Joffrey").getId(), 77.67397401574803);
+            expected.put(db.findNode(label, "name", "Robb").getId(), 73.56551401574802);
+            expected.put(db.findNode(label, "name", "Arya").getId(), 73.32532401574804	);
+            expected.put(db.findNode(label, "name", "Petyr").getId(), 72.26733401574802);
+            expected.put(db.findNode(label, "name", "Sansa").getId(), 71.56470401574803);
+            tx.success();
+        }
+    }
+
+    @Parameterized.Parameters(name = "{0}")
+    public static Collection<Object[]> data() {
+        return Arrays.asList(
+                new Object[]{"Heavy"},
+                new Object[]{"Light"},
+                new Object[]{"Kernel"},
+                new Object[]{"Huge"}
+        );
+    }
+
+    @Parameterized.Parameter
+    public String graphImpl;
+
+    @Test
+    public void testStream() throws Exception {
+        final Map<Long, Double> actual = new HashMap<>();
+        runQuery(
+                "CALL algo.eigenvector.stream('Character', 'INTERACTS_SEASON1', {graph:'"+graphImpl+"', direction: 'BOTH'}) " +
+                        "YIELD nodeId, score " +
+                        "RETURN nodeId, score " +
+                        "ORDER BY score DESC " +
+                        "LIMIT 10",
+                row -> actual.put(
+                        (Long)row.get("nodeId"),
+                        (Double) row.get("score")));
+
+        assertMapEquals(expected, actual);
+    }
+
+//    @Test
```

Comment:
can we delete these or do you want to keep them for weighted EV ?

Reply:
yeh I was keeping them so that when I add weighted EV I'll uncomment them